ui/espresso/BasicSample/app/src/main/EmptyManifest.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The file name says empty but it's no longer empty

Reply:
Same for the rest of the manifest files

Reply:
Done, renamed to AppManifest.xml